src/flutter/shell/platform/linux_embedded/surface/environment_egl_drm_eglstream.h

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_ENVIRONMENT_EGL_DRM_EGLSTREAM_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_ENVIRONMENT_EGL_DRM_EGLSTREAM_H_
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+
+#include <cstring>
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/surface/environment_egl.h"
+
+namespace flutter {
+
+class EnvironmentEglDrmEglstream : public EnvironmentEgl<void> {
+ public:
+  EnvironmentEglDrmEglstream(int drm_device) : EnvironmentEgl<void>() {
```

Comment:
EGLint

Reply:
Since this argument becomes unnecessary, I removed it.